scripts/screener/screener.types.ts

```diff
@@ -90,12 +90,17 @@ export interface ScreenerStepBuilder {
 /** Keys of `themes` object exported from `@fluentui/react-northstar/src/index`. */
 export type ScreenerThemeName = 'teams' | 'teamsDark' | 'teamsHighContrast' | 'teamsV2' | 'teamsDarkV2';
 
+export type ScreenerTheme = {
+  name: ScreenerThemeName;
+  testResultName?: ScreenerThemeName;
```

Comment:
```suggestion
  /* 
   * Decouples the test name from the theme name.
   * Useful to remap tests to different themes to avoid generating new screenshots
   */
  testResultName?: ScreenerThemeName;
```